pkg/audit/containment_test.go

```diff
@@ -12,14 +12,18 @@ import (
 	"storj.io/storj/internal/testplanet"
 	"storj.io/storj/internal/testrand"
 	"storj.io/storj/pkg/audit"
+	"storj.io/storj/pkg/overlay"
 	"storj.io/storj/pkg/pkcrypto"
 	"storj.io/storj/pkg/storj"
 )
 
 func TestContainIncrementAndGet(t *testing.T) {
 	testplanet.Run(t, testplanet.Config{
-		SatelliteCount: 1, StorageNodeCount: 4,
+		SatelliteCount: 1, StorageNodeCount: 2,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		containment := planet.Satellites[0].DB.Containment()
+		cache := planet.Satellites[0].DB.OverlayCache()
```

Comment:
I would move these two declarations before the line where they are first used.

Reply:
I initialize these vars at the beginning to have a clearer code below. If your preference is to use them inline the main code logic then I would not use them at all.

Reply:
I've been browsing code, I've found some places where vars were defined before they were needed and it required me to go  up to see what they were.

But I understand your point considering that this is a test.

Another note, I prefer when I have those group initialization to use a `var` block, because it makes clearer the initialization purpose.